include/af/lapack.h

```diff
@@ -200,6 +200,26 @@ namespace af
     */
     AFAPI array inverse(const array &in, const matProp options = AF_MAT_NONE);
 
+#if AF_API_VERSION >= 37
+    /**
+       C++ Interface for pseudo-inverting (Moore-Penrose) a matrix.
+       Currently uses the SVD-based approach.
+
+       \param[in] in is the input matrix
+       \param[in] tol defines the lower threshold for singular values from SVD
+       \param[in] options determining various properties of matrix \p in
+       \returns \p x, the inverse of the input matrix
```

Comment:
I don't see an x value in the function signature. just use
\returns the inverse of the input matrix